doc/specifications/proposed/translations/removal.md

```diff
@@ -0,0 +1,55 @@
+# Removal of Content Object Translations
+
+## Deleting single translation from a content object
+
+Sometimes there is a need to delete single translation of a content object without deleting
+the object itself.
+
+To aid that, PHP API introduces the `deleteTranslation` method on `ContentService` specified as:
+
+```php
+/**
+ * Delete content object translation by creating new version draft without the specified translation.
+ *
+ * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
+ *         is the only one content object has.
+ *
+ * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
+ * @param string $languageCode
+ *
+ * @return \eZ\Publish\API\Repository\Values\Content\Content Content draft
+ */
+public function deleteTranslation(VersionInfo $versionInfo, $languageCode);
+```
+
+Content object with deleted translation is a new version of that object, thus `deleteTranslation`
+returns draft to be published.
+
+## Removing specific language translation from all the content object versions
+
+For the cases of system maintenance involving e.g. removing a language, PHP API introduces
+the `purgeTranslations` method on `ContentService`, specified as:
+
+```php
+/**
+ * Purge content object translation from all versions (including archived ones) of a content object.
+ *
+ * NOTE: this operation is risky and permanent, so user interface should provide a warning before
+ *       performing it.
+ *
+ * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
+ *         is the only one content object has.
+ *
+ * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+ * @param string $languageCode
+ *
+ */
+public function purgeTranslations(ContentInfo $contentInfo, $languageCode);
```

Comment:
Also naming... Maybe, unsure.. There is no use of the word purge in the API. So more consistent would be delete or remove, unless you see this as a different case then the other maybe?